src/main/java/com/mastery/java/task/config/AppConfiguration.java

```diff
@@ -1,4 +1,22 @@
 package com.mastery.java.task.config;
 
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+
+import javax.sql.DataSource;
+
+@Configuration
 public class AppConfiguration {
+
+    @Bean
+    public DataSource getDataSource() {
+        DriverManagerDataSource ds = new DriverManagerDataSource();
+        ds.setDriverClassName("org.postgresql.Driver");
+        ds.setUrl("jdbc:postgresql://localhost/employeedb");
+        ds.setUsername("postgres");
+        ds.setPassword("12345");
+        return ds;
```

Comment:
Use application.properties file to store DB credentials and @Value to get it in the AppConfiguration class instead hardcode it.

Reply:
Done